### PR TITLE
WIP: olha/k8s/20 backend/21 redis/22 mongodb

### PR DESCRIPTION
**Pull request by KhOlga**

### CODE REVIEW

File: K8s_README.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+kubectl apply -f .\mongodb\
+```
+
+Next step: deploy backend application:
+```
+kubectl apply -f .\backend\
+```
+
+Next step: setup value for NEXT_PUBLIC_API_URL in `.\frontend\01_frontend_config_map_secrets.yml`
+
```
Comment (AISlearning) [inline K8s_README.md]:
```suggestion
Tip: run kubectl get svc --namespace=development and retrieve the external ip from backend. This is the value of NEXT_PUBLIC_API_URL
```

File: kubernetes/mongodb/30_service.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: mongodb-service
```
Comment (AISlearning) [inline kubernetes/mongodb/30_service.yml:4]:
```suggestion
  name: mongo
```

File: kubernetes/mongodb/20_deployment.yml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  selector:
+    matchLabels:
+      app: mongodb
+  template:
+    metadata:
+      labels:
+        app: mongodb
+    spec:
+      containers:
+        - name: mongodb
```
Comment (AISlearning) [inline kubernetes/mongodb/20_deployment.yml]:
```suggestion
        - name: mongo
```

File: kubernetes/redis/10_deployment.yml [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+kind: Deployment
+metadata:
+  name: redis-deployment
+  namespace: development
+  labels:
+    app: redis
+#    app.kubernetes.io/created-by: team-1
+#    app.kubernetes.io/type: app
+#    app.kubernetes.io/name: redis
+#    app.kubernetes.io/component: database
```
Comment (AISlearning) [inline kubernetes/redis/10_deployment.yml:11]:
```suggestion
```

File: kubernetes/redis/20_service.yml [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+    app: redis
+spec:
+  type: ClusterIP
+  ports:
+    - port: 6379
+      targetPort: 6379
+  selector:
+#    app.kubernetes.io/name: redis
+#    app.kubernetes.io/instance: redis
+#    app.kubernetes.io/component: service
```
Comment (AISlearning) [inline kubernetes/redis/20_service.yml]:
```suggestion
```

**Review by AISlearning — CHANGES_REQUESTED** [review]
Can you apply the changes and remove all folders with old in it? Also can you remove 40, 50 and 60 from mongo and also can you remove one of the 10 files from frontend? I have both configs and config-map fro frontend

**Comment by KhOlga** [comment]
I removed folders with 'old'.

But why can't we keep mongo-express?
40, 50, 60 for mongo-express

**Review by AISlearning — APPROVED** [review]

**Pull request merged**